pkgs/applications/audio/renoise/beta.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, requireFile, fetchurl, libX11, libXext, libXcursor, 
+  libXrandr, libjack2, alsaLib, personal_sha256, 
+  ... }:
+
+stdenv.mkDerivation rec {
+  name = "renoise_" + version ;
+  version = "beta_4";
+
+  buildInputs = [ libX11 libXext libXcursor libXrandr alsaLib libjack2 ];
+
+  src = let name = if builtins.currentSystem == "x86_64-linux" then
+                     "rns_3_1_0b4_linux_x86_64.tar.gz"
+                   else 
+                     "rns_3_1_0b4_linux_x86.tar.gz";
+            message = ''
+              please go to http://backstage.renoise.com/frontend/app/index.html#/login and download the binary.
+              import the binary by : nix-store --add-fixed sha256 ${name}
+              find out the hash by : shasum -a 256 ${name}
+              and set the hash in your /etc/nixos/configuration.nix or ~/.nixpkgs/config.nix
```

Comment:
Oh, sorry, I see that you've provided it. IMO we have to manually set hash each time it've changed. So the option to set it by user is not good. 
@vcunat what do you think?